VENTokenField/VENBackspaceTextField.m

```diff
@@ -24,15 +24,12 @@
 
 @implementation VENBackspaceTextField
 
-- (BOOL)keyboardInputShouldDelete:(UITextField *)textField
-{
+- (void)deleteBackward {
     if (self.text.length == 0) {
         if ([self.backspaceDelegate respondsToSelector:@selector(textFieldDidEnterBackspace:)]) {
             [self.backspaceDelegate textFieldDidEnterBackspace:self];
         }
     }
-
-    return YES;
 }
```

Comment:
`keyboardInputShouldDelete` seemed to prevent `textField:shouldChangeCharactersInRange:replacementString` from being called on backspaces. `deleteBackward` doesn't seem to have the same limitation